### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by jakemwood**

### CODE REVIEW

**Review by lorimonteleone — APPROVED** [review]
🎉

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
...
+[![Python package](https://github.com/TestBoxLab/chalice-api/actions/workflows/test.yml/badge.svg)](https://github.com/TestBoxLab/chalice-api/actions/workflows/test.yml)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+**Chalice x APISpec x Pydantic plug-ins**
+
+Combines the power of Chalice, APISpec, and Pydantic to make AWS Chalice apps easily documented
+
+## Installation
+
+First, add chalice-api:
```
Comment (n1tsua) [inline README.md:12]: should we call this repo/library `chalice-pydantic-apispec`?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+```python
+@app.route('/', methods=["POST"], post_body_model=MySchema)
+def example():
+    body = MySchema.parse_obj(app.current_request.json_body)
+```
+
+If you have multiple methods supported, you may have something like:
+
+```python
+@app.route('/', methods=["POST", "PUT"], post_body_model=MyCreateSchema, put_body_model=MyEditSchema)
```
Comment (n1tsua) [inline README.md]: post_body_model -> post_req_schema ?
put_body_model -> put_req_schema ?

Would it be possible to instead do something like 

`@app.route('/', methods={"POST": (MyCreateReqSchema, MyCreateResSchema), "PUT": (MyUpdateReqSchema, MyUpdateResSchema)})`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,80 @@
...
+    # code goes here
+    pass
+```
+
+### Responses
+
+```python
+@app.route("/", methods=["POST", "PUT"],
+           post_response_model=MyCreateResponseSchema,
+           put_response_body=MyEditResponseSchema)
```
Comment (n1tsua) [inline README.md]: Is this a typo? post_response_model and put_response_**body**

File: chaliceapi/__init__.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                    }
+                }
+            }
+        }
+
+        return operations
+
+    def _handle_response(self, spec, method, kwargs):
+        # Now the response!
+        model_name = f"{method}_response_body"
```
Comment (n1tsua) [inline chaliceapi/__init__.py]: Mentioned this previously, but can we standardize these names pls

File: chaliceapi/__init__.py [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+            operations = {}
+            for method in methods:
+                operations = {
+                    **operations,
+                    **self._handle_request(spec, method, kwargs),
+                }
+                operations = {
+                    **operations,
+                    **self._handle_response(spec, method, kwargs),
+                }
```
Comment (n1tsua) [inline chaliceapi/__init__.py]: how very javascript of you

**Review by n1tsua — APPROVED** [review]

**Pull request merged**